pages/integrations/amazon_eventbridge.md

```diff
@@ -449,6 +449,8 @@ AWS EventBridge has strict limits on the size of the payload as documented in [A
   }
 }
 ```
+>📘
+> To determine if a job has <code>passed</code> or <code>failed</code>, that is reflected by the "passed" boolean which is <code>true</code> or <code>false</code>.
```

Comment:
I think we can simplify the language a bit, and use Markdown formatting. Thinking:

```suggestion
> To check a job's result, use the `passed` field. The value is `true` when the job passed, and `false` otherwise.
```

What do you think?